spec/requests/distributions_summary_requests_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.describe "Distributions", type: :request do
+  context "while signed in" do
+    before do
+      sign_in(@user)
+    end
+
+    describe "GET #index" do
+      it "shows a list of recent distributions" do
+        get distributions_summary_path(@user.organization)
+        expect(response.body).to include("Recent distributions")
```

Comment:
Should we do any additional checks here? E.g. create a distribution and ensure its name is in the body?